mscore/plugin/qmlpluginengine.cpp

```diff
@@ -48,7 +48,8 @@ void QmlPluginEngine::beginEndCmd(MuseScore* ms, bool inUndoRedo)
             undoRedo = true;
 
       const Score* cs = ms->currentScore();
-      currScoreState = cs->masterScore()->state(); // score and excerpts have united undo stack so we are better to track master score
+      if (cs)
+            currScoreState = cs->masterScore()->state(); // score and excerpts have united undo stack so we are better to track master score
```

Comment:
Oh, then it's a regression from my #5626.
It is better then to fix it like
```
currScoreState = cs ? cs->masterScore()->state() : ScoreContentState();
```
as this variable should better always reflect the actual state.

Reply:
Done.